src/Fantomas.Analyzers/Fantomas.Analyzers.fsproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.Build.NoTargets/1.0.80"> <!-- This is not a project we want to build. -->
```

Comment:
Not a fan of putting this in the source.
We should have this somewhere in the repo root.
Eventually, we will start writing Fantomas-specific analyzers which should be in this project.
Nit: the filename can just be `analyzers.proj`